deploy/prod-nci/frontend-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```

Comment:
I don't see any changes between this file and the original file in deploy/kubernetes. If there are no changes, you should use the yaml file, so this one can be removed.

Reply:
I have removed the deployment file